src/plugins/credential_management/server/routes/create.ts

```diff
@@ -24,11 +24,11 @@ export function registerCreateRoute(router: IRouter) {
                   schema.literal('basic_auth'),
                   schema.literal('aws_iam_credential'),
                 ]),
-                basic_auth_credential_JSON: schema.maybe(schema.object({
+                basic_auth_credential_json: schema.maybe(schema.object({
```

Comment:
weird name to me,
how about name it `username_password_credential`
no need to add json postfix